src/Tile/_TileMedia.js

```diff
@@ -3,21 +3,31 @@ import PropTypes from 'prop-types';
 import React from 'react';
 
 const TileMedia = props => {
-    const { children, className, ...rest } = props;
+    const { children, className, productTile, backgroundImage, ...rest } = props;
 
     const tileMediaClasses = classnames(
-        'fd-tile__media',
+        {
+            'fd-tile__media': !productTile,
+            'fd-product-tile__media': productTile
+        },
         className
     );
 
-    return <div {...rest} className={tileMediaClasses}>{children}</div>;
+    return (
+        <div {...rest} className={tileMediaClasses}
+            style={{ backgroundImage: 'url(' + backgroundImage + ')' }}>{children}</div>);
 };
 
 TileMedia.displayName = 'Tile.Media';
 
 TileMedia.propTypes = {
+    backgroundImage: PropTypes.string,
```

Comment:
I was thinking this prop would/could be on the `Tile` component since that already has a prop called `backgroundColor`.  To me, `TileMedia` is more designed as a wrapper for children (images) rather than a URL string prop.

Reply:
@greg-a-smith would you have the style tag for backgroundImage go on Tile itself then instead of media or would you pass down the backroundImage prop to media?  When I originally put it on Tile, it did not format correctly: the image took up the entire background of the Title portion as well and was not big enough, so thought to keep it in Media where there is the 'fd-product-tile__media' class has previously lived maybe making it easier for the consumer to associate media with a photo in general? (whether it be background of small icon). 

Either if fine with me - can you confirm if you want it on tile I should put the style tag on tile and add the fd-product-tile__media class name as well?

Reply:
I think the final emitted markup would be the same as you have it now, but `Tile` would have the prop and would render the child `<div>` where it needs it to go.  You are right, the way the styles are written, placing the `style` attribute on the top-level tile `<div>` would not be correct.